tests/test_deprecated.py

```diff
@@ -58,6 +58,23 @@ def test_tbd_remove_in_v0_9_0_module_imports():
     from pytorch_lightning.logging.wandb import WandbLogger  # noqa: F402
 
 
+def test_tbd_remove_in_v0_9_0_trainer():
+    mapping_old_new = {
+        'fast_dev_run': 'unit_test',
+    }
+    # skip 0 since it may be interested as False
+    kwargs = {k: (i + 1) for i, k in enumerate(mapping_old_new)}
+
+    trainer = Trainer(**kwargs)
+
+    for attr_old in mapping_old_new:
+        attr_new = mapping_old_new[attr_old]
+        assert kwargs[attr_old] == getattr(trainer, attr_old), \
+            'Missing deprecated attribute "%s"' % attr_old
+        assert kwargs[attr_old] == getattr(trainer, attr_new), \
+            'Wrongly passed deprecated argument "%s" to attribute "%s"' % (attr_old, attr_new)
```

Comment:
as it become duplicated, move this to a particular function `_aasert_trainer_atribs(...)`

Reply:
I have made the requested changes. Please review whenever possible